### PR TITLE
First version of GHA CI script for Myanmar tools

### PR DESCRIPTION
**Pull request by gnrunge**
Migrated selected tasks from Travis CI.

### CODE REVIEW

File: .github/workflows/mynmr_tools_ci.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          java-version: '8'
+      - name: Consistency genconvert vs compiled
+        run: |
+          make transcompile
+          # There should not be any dirty files:
+          git update-index --refresh
+          git diff-index HEAD
+
+  # Java Client
+  compile-and-test-java-client:
```
Comment (sffc) [inline .github/workflows/mynmr_tools_ci.yml:39]:
```suggestion
  java:
```

File: .github/workflows/mynmr_tools_ci.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        with:
+          java-version: '7'
+      - name: Java client test
+        run: |
+          cd clients/java
+          mvn compile
+          mvn test
+      
+  # Python Client
+  python-client-test:
```
Comment (sffc) [inline .github/workflows/mynmr_tools_ci.yml:52]:
```suggestion
  python:
```

File: .github/workflows/mynmr_tools_ci.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        with:
+          java-version: '8'
+      - name: Training Data
+        run: |
+          cd training
+          mvn compile
+          mvn test
+
+  # Check consistency between genconvert and compiled converters
+  consistency-check-genconvert-vs-compiled-converter:
```
Comment (sffc) [inline .github/workflows/mynmr_tools_ci.yml:25]:
```suggestion
  genconvert:
```

File: .github/workflows/mynmr_tools_ci.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: '*'
+
+jobs:
+
+  # Training Data Project
+  training-data:
```
Comment (sffc) [inline .github/workflows/mynmr_tools_ci.yml:12]:
```suggestion
  training:
```

**Review by sffc — COMMENTED** [review]
Thank you!

I listed some suggestions for the job IDs.  The job names are where we can put longer descriptions, but I think it's nice to have short IDs.

File: .github/workflows/mynmr_tools_ci.yml (hunk 1 above)
Comment (gnrunge) [reply .github/workflows/mynmr_tools_ci.yml]: Done.

File: .github/workflows/mynmr_tools_ci.yml (hunk 2 above)
Comment (gnrunge) [reply .github/workflows/mynmr_tools_ci.yml]: Done.

File: .github/workflows/mynmr_tools_ci.yml (hunk 3 above)
Comment (gnrunge) [reply .github/workflows/mynmr_tools_ci.yml]: Done.

File: .github/workflows/mynmr_tools_ci.yml (hunk 4 above)
Comment (gnrunge) [reply .github/workflows/mynmr_tools_ci.yml]: Done.

**Comment by echeran** [comment]
Looks good so far. Assuming that you're going top-to-bottom in .travis.yml when converting to the Github Actions workflow, it looks like you skipped over the C++ client job (that theoretically might go between the `genconvert` and `java` jobs). Is that intentional?

**Review by sffc — APPROVED** [review]
Good to start, and I filed #71 to finish migrating the rest of the targets (which are all important).

**Comment by gnrunge** [comment]
>
> Looks good so far. Assuming that you're going top-to-bottom in .travis.yml
> when converting to the Github Actions workflow, it looks like you skipped
> over the C++ client job (that theoretically might go between the
> genconvert and java jobs). Is that intentional?
>
I started with the tasks I thought of as easiest to convert. I went
top-to-bottom but, thus, skipped some.

**Pull request merged**